patch-package
--- a/node_modules/vue-steemconnect/dist/index.js
+++ b/node_modules/vue-steemconnect/dist/index.js
@@ -4,9 +4,9 @@ Object.defineProperty(exports, "__esModule", {
   value: true
 });
 
-var _sc2Sdk = require('sc2-sdk');
+var _sc2Sdk = require('steemconnect');
 
-var sc2 = require('sc2-sdk');
+var sc2 = require('steemconnect');
 
 
 var VueSteemConnect = {
@@ -33,7 +33,8 @@ var VueSteemConnect = {
       namespaced: true,
       state: {
         user: null, // steemconnect user
-        accessToken: null // steemconnect access token
+        accessToken: null, // steemconnect access token
+		stdLogin: false //identifies login type
       },
       getters: {
         user: function user(state) {
@@ -41,7 +42,10 @@ var VueSteemConnect = {
         },
         accessToken: function accessToken(state) {
           return state.accessToken;
-        }
+        },
+		stdLogin: function stdLogin(state) {
+		  return state.stdLogin;
+		}
       },
       mutations: {
         login: function login(state, user) {
@@ -52,15 +56,24 @@ var VueSteemConnect = {
         },
         setAccessToken: function setAccessToken(state, accessToken) {
           state.accessToken = accessToken;
-        }
+        },
+		setStdLogin: function setStdLogin(state, stdLogin) {
+          state.stdLogin = stdLogin;
+        },
       },
       actions: {
-        login: function login(_ref) {
+        login: async function login(_ref) {
           var commit = _ref.commit,
               dispatch = _ref.dispatch,
               state = _ref.state;
 
-          return new Promise(function (resolve, reject) {
+          return new Promise(async function (resolve, reject) {
+			//take into consideration stdlogin case:
+			
+			let std_login = localStorage.getItem('std_login')
+			console.log('found std_login LS:'+std_login);
+			commit('setStdLogin', true);
+			
             // don't do anything if user data is already set
             if (!state.user) {
               // in that case we look for an access token in localStorage
@@ -68,14 +81,45 @@ var VueSteemConnect = {
               if (accessToken) {
                 // set access token and try to fetch user object
                 Vue.SteemConnect().setAccessToken(accessToken);
-                Vue.SteemConnect().me(function (err, user) {
-                  if (err) reject(err);else {
-                    // save user object in store
-                    commit('login', user);
-                    commit('setAccessToken', accessToken);
-                    resolve();
-                  }
-                });
+				//grab localstorage val to see if SC login used
+				
+				if (!std_login){
+					Vue.SteemConnect().me(function (err, user) {
+					  if (err) reject(err);else {
+						// save user object in store
+						commit('login', user);
+						commit('setAccessToken', accessToken);
+						commit('setStdLogin', false);
+						resolve();
+					  }
+					});
+				}else{
+					let username = localStorage.getItem('std_login_name');
+					let url = new URL(process.env.actiAppUrl + 'fetchUserData/?user='+username);
+			
+					let reqHeads = new Headers({
+					  'Content-Type': 'application/json',
+					  'x-acti-token': 'Bearer ' + accessToken,
+					});
+					let res = await fetch(url, {
+						headers: reqHeads
+					});
+					let outcome = await res.json();
+					//console.log(outcome);
+					if (outcome.error || !outcome.success){
+						reject(outcome.error)
+					}else{
+						//attempt to auth and grab user data
+						let userSC = new Object();
+						userSC.account = outcome.userdata;
+						console.log('success stdLogin');
+						// save user object in store
+						commit('setStdLogin', true);
+						commit('login', userSC)
+						commit('setAccessToken', accessToken);
+						resolve();
+					}
+				}
               }
             }
           });
@@ -85,7 +129,10 @@ var VueSteemConnect = {
 
           // remove access token and unset user in store
           localStorage.removeItem('access_token');
+		  localStorage.removeItem('std_login');
+		  localStorage.removeItem('std_login_name');
           commit('logout');
+		  commit('setStdLogin', false);
           commit('setAccessToken', null);
         }
       }
--- a/node_modules/vue-steemconnect/src/index.js
+++ b/node_modules/vue-steemconnect/src/index.js
@@ -1,5 +1,5 @@
-const sc2 = require('sc2-sdk');
-import { Initialize } from 'sc2-sdk';
+const sc2 = require('steemconnect');
+import { Initialize } from 'steemconnect';
 
 const VueSteemConnect = {
   install(Vue, _options) {
@@ -39,6 +39,9 @@ const VueSteemConnect = {
         login (state, user) {
           state.user = user
         },
+		loginStd (state, user) {
+		  state.user = user
+		},
         logout (state) {
           state.user = null
         },
@@ -50,21 +53,32 @@ const VueSteemConnect = {
         login ({ commit, dispatch, state }) {
           return new Promise((resolve, reject) => {
             // don't do anything if user data is already set
+			console.log(state.user);
             if (!state.user) {
               // in that case we look for an access token in localStorage
               const accessToken = localStorage.getItem('access_token')
               if (accessToken) {
+				console.log('SC plugin in');
                 // set access token and try to fetch user object
                 Vue.SteemConnect().setAccessToken(accessToken)
-                Vue.SteemConnect().me((err, user) => {
-                  if (err) reject(err)
-                  else {
-                    // save user object in store
-                    commit('login', user)
-                    commit('setAccessToken', accessToken)
-                    resolve()
-                  }
-                })
+				if (!this.$store.state.stdLoginUser){
+					Vue.SteemConnect().me((err, user) => {
+					  if (err) reject(err)
+					  else {
+						// save user object in store
+						commit('login', user)
+						commit('setAccessToken', accessToken)
+						resolve()
+					  }
+					})
+				}else{
+					// save user object in store
+					let userSC = new Object();
+					userSC.account = json.userdata;
+					commit('login', userSC)
+					commit('setAccessToken', accessToken)
+					resolve()
+				}
               }
             }
           })
@@ -72,9 +86,13 @@ const VueSteemConnect = {
         logout ({ commit }) {
           // remove access token and unset user in store
           localStorage.removeItem('access_token')
+		  localStorage.removeItem('std_login')
           commit('logout')
           commit('setAccessToken', null)
-        }
+        },
+		loginStd ({ commit }) {
+		  commit('login', user)
+		}
       }
     }
   }
